mpas_analysis/sea_ice/modelvsobs.py

```diff
@@ -160,21 +168,23 @@ def seaice_modelvsobs(config, streamMap=None, variableMap=None):
     print "  Load sea-ice data..."
     ds = open_multifile_dataset(fileNames=fileNames,
                                 calendar=calendar,
+                                simulationStartTime=simulationStartTime,
                                 timeVariableName='Time',
                                 variableList=['iceAreaCell',
                                               'iceVolumeCell'],
                                 variableMap=variableMap,
                                 startDate=startDate,
-                                endDate=endDate,
-                                yearOffset=yearOffset)
+                                endDate=endDate)
 
     # Compute climatologies (first motnhly and then seasonally)
     print "  Compute seasonal climatologies..."
-    timeStart = datetime.datetime(yearOffset+startYear, 1, 1)
-    timeEnd = datetime.datetime(yearOffset+endYear, 12, 31)
-    dsTimeSlice = ds.sel(Time=slice(timeStart, timeEnd))
-    # check that each year has 24 months (?)
-    monthlyClimotology = dsTimeSlice.groupby('Time.month').mean('Time')
+
+    months = [date.month for date in days_to_datetime(ds.Time,
+                                                      calendar=calendar)]
+
+    ds.coords['month'] = ('Time', months)
+    monthlyClimatology = ds.groupby('month').mean('Time')
+
     daysInMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
     monthLetters = ['J', 'F', 'M', 'A', 'M', 'J', 'J', 'A', 'S', 'O', 'N', 'D']
```

Comment:
We really should have / use these variables from the constants file location: https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/shared/constants/constants.py.  I don't think we should just hard-code this type of information in these files, but this is out of scope for this PR.